controllers/replication-controller/dellcsireplicationgroup_controller.go

```diff
@@ -372,6 +372,18 @@ func (r *ReplicationGroupReconciler) processSnapshotEvent(ctx context.Context, g
 		return err
 	}
 
+	if _, err := remoteClient.GetNamespace(ctx, actionAnnotation.SnapshotNamespace); err != nil {
```

Comment:
Will there be a prerequisite to create snapshot namespace? Or only the create_snapshot action creates the namespace?

Reply:
This satisfies the prerequisite of creating the namespace ahead of time. Using the `create snapshot` action will create the namespace if it doesn't exist so the user doesn't have to bounce between clusters to figure that out.